main/views.py

```diff
@@ -56,11 +56,7 @@ def home(request):
 
     metrics.gui_home_page_hits.inc()
 
-    non_empty_datasets = (
-        RefData.objects.values_list('dataset', flat=True).
-        distinct())
-
-    total_indexed_citations = RefData.objects.count()
+    total_indexed_citations = estimate_count('api_ref_data')  # Use estimate
```

Comment:
This is not cached, right?
This shows the string: `268.58k bibliographic items now indexed`.
IMO we don't have to show this on the home page.
If it's hurting the database, I'm in favour of taking that line out.

Reply:
I don't think it's cached. And yes, that's what is used to build that message. 

The new query should run very fast, but it's an estimate, not an exact count. If that's good enough, I'm happy to keep it.

Reply:
I still think it's a usless statistic to open a db connection for. Specially on the home page.

Reply:
I am inclined to agree that this is noise for the UI - we should remove the line altogether unless there's no other way to discover what the datatsets are.

Reply:
As per the discussion earlier, I have removed the query for getting the row count of the table, but kept the dataset links. It should now say

> Bibliographic items are indexed among sources: rfcs, ids, rfcsubseries, misc, w3c, 3gpp, ieee, iana, nist